rules/Multifactor-Google-Authenticator-Do-Not-Rename.js

```diff
@@ -1,35 +1,74 @@
 function (user, context, callback) {
-
-    var mfa_enabled_connection = [
+    var AUTHENTICATOR_LABEL = 'MOJ Analytical Platform (Alpha)';
+    var DISABLED_CLIENTS = [
+        'oUb1V330oXKyMpTagAYDzWDY10U4ffWF', // kubectl-oidc
+    ];
+    var ENABLED_CONNECTIONS = [
         'github',
-        'google-oauth2'
-    ].indexOf(context.connection) !== -1;
+        'google-oauth2',
+    ];
+    var MFA_CHALLENGE_EVERY_MINUTES = 8 * 60; // 8 hours
 
-    // Exclude the following clients from mfa
-    var mfa_disabled_clients = [
-        'oUb1V330oXKyMpTagAYDzWDY10U4ffWF'
-    ].indexOf(context.clientID) === -1;
 
-    var user_with_mfa = user.app_metadata && user.app_metadata.use_mfa;
+    var enabled_for_user = !user.app_metadata || user.app_metadata.use_mfa !== false;
+    var enabled_for_connection = ENABLED_CONNECTIONS.indexOf(context.connection) !== -1;
+    var enabled_for_client = DISABLED_CLIENTS.indexOf(context.clientID) === -1;
 
-    if (
-        mfa_enabled_connection &&
-        mfa_disabled_clients
-    ) {
+    if ( !enabled_for_user || !enabled_for_connection || !enabled_for_client ) {
```

Comment:
minor thing, but this might be nicer if you invert the logic and have `if (disabled_for_user || disabled_for_connection ...`

Reply:
Also a minor thing, but [Do not add spaces inside parentheses](https://github.com/airbnb/javascript#whitespace--in-parens)